src/main.js

```diff
@@ -0,0 +1,20 @@
+function daysInMonth(month, year) {
+  try {
+    if (!Number.isInteger(year) || !month || !Number.isInteger(month) || month > 12) {
```

Comment:
`!month` interesting but if we call the function with `daysInMonth(-2, 2019);`?

Reply:
done